doc/source/new-project-guide/level-5-tutorial.rst

```diff
@@ -218,13 +218,13 @@ We require that each PR includes a news item as a ``<branch-name>.rst`` file und
         git checkout skpkg-migration
         git pull origin skpkg-migration
 
-#. Make a copy of ``news/TEMPLATE.rst`` and rename to ``news/<branch-name>.rst``. We receommend use the following command to streamline this process.
+#. Make a copy of ``news/TEMPLATE.rst`` and rename to ``news/<branch-name>.rst``.
 
     .. code-block:: bash
```

Comment:
and also remove this code block.  We aren't recommending this approach to by  manually typed.

Reply:
yes, done